### PR TITLE
update service search document generation

### PR DESCRIPTION
**Pull request by PriyalMaheshwarii**
#### What type of PR is this?

- Improvement

#### What does this PR do / why is it needed ?
updated service search document to use elementPath as id

#### Does this PR introduce a user-facing change?
No

### CODE REVIEW

**Review by aormerod-gs — COMMENTED** [review]
Can we add a test for this in a future iteration

File: legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure [hunk 1]
```diff
@@ -157,7 +157,7 @@ function meta::analytics::search::transformation::mapping::buildPropertyList(pro
 function meta::analytics::search::transformation::service::buildDocument(service: meta::legend::service::metamodel::Service[1]): meta::analytics::search::metamodel::service::RootDocument[1]
 {
   ^meta::analytics::search::metamodel::service::RootDocument(
-    id = $service.pattern,
+    id = $service->elementToPath(),
```
Comment (aormerod-gs) [inline legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure:160]: This makes sense to be the ID, but should this really be extending from the packageable element root document and have a specific project for element path

**Review by aormerod-gs — APPROVED** [review]

File: legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure [hunk 2]
```diff
@@ -157,7 +157,7 @@ function meta::analytics::search::transformation::mapping::buildPropertyList(pro
 function meta::analytics::search::transformation::service::buildDocument(service: meta::legend::service::metamodel::Service[1]): meta::analytics::search::metamodel::service::RootDocument[1]
 {
   ^meta::analytics::search::metamodel::service::RootDocument(
-    id = $service.pattern,
```
Comment (aormerod-gs) [inline legend-engine-xt-analytics-search-pure/src/main/resources/core_analytics_search/trans.pure:160]: Can we add a test for this?

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**